media/video/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Upload module"""
+import logging
+from bson import ObjectId
+from flask import request
+import json
+from media import get_collection, blueprint
+from flask import Blueprint
+
+bp = Blueprint('project', __name__)
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, ObjectId):
+            return str(o)
+        return json.JSONEncoder.default(self, o)
+
+
+@bp.route('/project', methods=['POST'])
+def create_video_editor():
+    if request.method == 'POST':
+        files = request.files.to_dict(flat=False)['media']
```

Comment:
Just to keep attention ⚠️ , here might be a key error. This is something we will handle later by validation incoming data with http://docs.python-cerberus.org/en/stable/

Reply:
yes, we will use cerberus for validate the incoming data in upload file.